README.md

```diff
@@ -118,6 +118,56 @@ Example with `cifar10`:
 
  `pip install pycocotools`
 
+ ### Webdataset example
+
+Here is an example on how to run it on [webdatasets](https://github.com/webdataset/webdataset).
+First, you need to install `webdataset`.
+
+`pip install webdataset`
```

Comment:
what about we simply add it as dep ?

Reply:
We could do that: I was mostly keeping in line with the VTAB and TFDS examples, where it's installed separately.